test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
> This is not the proper place to discuss these issues.

I believe I already did, this is why I raised this question here

> I'm just following what we use everywhere else right now.

I'm not asking to change anything

> If you want to raise an issue with the current syntax

There is no current syntax, there is no format for it, it's just something we might use, this is why I created jquery/contribute.jquery.org#95 in the first place